src/poetry/core/__init__.py

```diff
@@ -5,7 +5,7 @@
 from pathlib import Path
 
 
-__version__ = "1.1.0a7"
+__version__ = "1.1.0b1"
```

Comment:
```suggestion
```

We have been promoting the use of importlib elsewhere for this use case. We should really dogfood our own advice on the matter and either remove this or use importlib/pkgutil.

Reply:
I'm not fussed either way - feel free either to update this MR or to reject it in favour of something else

Reply:
Let's see what the rest of the team thinks. My vote is to just remove it.

Reply:
Let's just drop it.

Reply:
Hmm, apparently not. We imprint the metadata for wheels with the version it seems.